docs/EN/operate/build.md

```diff
@@ -231,42 +266,94 @@ git fetch
 git pull --recurse
 ```
 
-If you see the word error or conflict on the output, head over to [Fix Customization Conflicts](#fix-customization-conflicts).
+The first command `git fetch` brings down any updates about the state of the Trio workspace repository from GitHub.
 
-Otherwise, you are ready to build.
+The second command `git pull --recurse` does several things:
 
-### Fix Customization Conflicts
+1. Brings down any updates about the state of the submodule repositories from GitHub, e.g., `Fetching submodule XXXX` repeated for each submodule
+2. Merges any changes from GitHub into your local copy of the Trio workspace
+3. For each submodule, updates the version to match that selected in GitHub
 
-If you have customizations in the Trio code (in the workspace) that cause a conflict when you pull, you can save and restore those with the `stash` and `stash pop` lines as shown below:
+There can be errors after step 2, e.g.,
 
+```{code-block}
+:class: no-copybutton
+error: Your local changes to the following files would be overwritten by merge:
+	FreeAPS/folder/file.swift
+Please commit your changes or stash them before you merge.
+Aborting
 ```
-git stash
-git fetch
+
+The `pull` command halts at this error and does not move on to step 3. You must first fix the Trio workspace files. 
+
+Head over to [Fix Trio Conflicts](#fix-trio-conflicts).
```

Comment:
I think merge this line into the previous line to get something like:

The `pull` command halts at this error and does not move on to step 3. You must first fix the Trio workspace files by following the instructions at [Fix Trio Conflicts](#fix-trio-conflicts).